adapter-cqld4/src/main/java/io/nosqlbench/cqlgen/binders/BindingsAccumulator.java

```diff
@@ -81,9 +81,8 @@ private Optional<Binding> resolveSymbolicBinding(CqlColumnBase def, Map<String,S
             Optional<Binding> binding = library.resolveBindingsFor(def);
             if (binding.isPresent()) {
                 if (binding.get().getRecipe()==null) {
-                    throw new RuntimeException("Binding returned from library " + library + "' was null, for def '" + def + "'. " +
-                        "This probably means you need to add a default binding for '" + def.getTrimmedTypedef() + "' to " +
-                    CGDefaultCqlBindings.DEFAULT_CFG_DIR+ File.separator+CGDefaultCqlBindings.DEFAULT_BINDINGS_FILE+" (see cqlgen help for details)");
+                    throw new RuntimeException(
+                            STR."Binding returned from library \{library}' was null, for def '\{def}'. This probably means you need to add a default binding for '\{def.getTrimmedTypedef()}' to \{CGDefaultCqlBindings.DEFAULT_CFG_DIR}\{File.separator}\{CGDefaultCqlBindings.DEFAULT_BINDINGS_FILE} (see cqlgen help for details)");
```

Comment:
Does it support multi line for better readability

Reply:
No